mxcubeweb/routes/login.py

```diff
@@ -45,15 +32,17 @@ def login():
         password = params.get("password", "")
 
         try:
-            res = jsonify(app.usermanager.login(login_id, password))
+            app.usermanager.login(login_id, password)
         except Exception as ex:
             msg = "[LOGIN] User %s could not login (%s)" % (
                 login_id,
                 str(ex),
             )
             logging.getLogger("MX3.HWR").exception("")
             logging.getLogger("MX3.HWR").info(msg)
-            res = deny_access("Could not authenticate")
+            res = make_response(jsonify({"msg": "Could not authenticate"}), 200)
+        else:
+            res = make_response(jsonify({"msg": ""}), 200)
```

Comment:
Why no message?

We could put the old message back in:

```suggestion
            res = make_response(jsonify({"msg": "Successful login"}), 200)
```

Reply:
FYI, if I'm not mistaken, the front-end has an equality check for the empty string, so it will need to be updated as well. This is temporary, though. The idea is to return the login info object directly eventually.

Reply:
I see, thanks! Then let's leave it like this for now, we only need to adjust the test.